### ISSUE TITLE
cljfmt has some aggressive ideas about comments

### ISSUE THREAD

**Issue by venantius** [weavejester/cljfmt#14]
Running `lein cljfmt check` on [Ultra](https://github.com/venantius/ultra), I hit this pleasant little critter:

```clojure
-  #_(if (not (false? tests))
-    (test/activate!))
+  #_[:tag :list][:format-string "(%s)"][:wrap-length 2][:seq-fn #<seq$list_node$fn__6552 rewrite_clj.node.seq$list_node$fn__6552@ab4d2ba>][:children (<token: if> <whitespace: " "> <list: (not (false? tests))> <newline: "\n"> <list: (test/activate!)>)]
```

Suffice it to say that this seems...probably unintended. Thoughts?

**Comment by weavejester** [comment]
This should be fixed in cljfmt 0.1.4 (see #11). Which version are you using?

**Comment by venantius** [comment]
Ah, great - I was on 0.1.2. Upgrading to 0.1.4 does fix it.
